lms/services/hubspot/_client.py

```diff
@@ -18,6 +35,66 @@ def get_companies(self):
         ]
         yield from self._get_objects(self._api_client.crm.companies, fields)
 
+    def create_company_import(
+        self, contract_year_billables: list[tuple[str, int, int]]
+    ):
+        """Start an import data from a CSV file to Hubspot to update companies.
+
+        :param contract_year_billables: A list of tuples with: (company id in HubSpot, unique teachers, unique users)
+        """
+        with NamedTemporaryFile(mode="w", suffix=".csv") as csv_file:
+            writer = csv.writer(csv_file)
+            for row in contract_year_billables:
+                writer.writerow(row)
+            # Ensure all rows are written to disk before we start to upload
+            csv_file.flush()
+
+            files = [
+                {
+                    "fileName": os.path.basename(csv_file.name),
+                    "fileFormat": "CSV",
+                    "fileImportPage": {
+                        "hasHeader": False,
+                        "columnMappings": [
+                            {
+                                "columnObjectTypeId": ObjectType.COMPANY,
+                                "columnName": "hs_object_id",
+                                "propertyName": "hs_object_id",
+                                "idColumnType": "HUBSPOT_OBJECT_ID",
+                            },
+                            {
+                                "columnObjectTypeId": ObjectType.COMPANY,
+                                "columnName": "billable_teachers_this_contract_year",
+                                "propertyName": "billable_teachers_this_contract_year",
+                                "idColumnType": None,
+                            },
+                            {
+                                "columnObjectTypeId": ObjectType.COMPANY,
+                                "columnName": "billable_users_this_contract_year",
+                                "propertyName": "billable_users_this_contract_year",
+                                "idColumnType": None,
+                            },
+                        ],
+                    },
+                }
+            ]
+
+            import_request = {
+                "name": f"contract_year_import_{date.today().isoformat()}",
```

Comment:
I think the dates should probably be part of the `contract_year_billables` argument, rather than using the current day here. Those numbers of unique teachers and users in `contract_year_billables` were calculated at to a given date, it'd be better to include that date in the data rather than implicitly assuming that it matches `date.today()` at the time when this line later runs.

Reply:
I have included this as a second independent parameter